tests/pkcli/fmt_data/check1.out/pkexcept

```diff
@@ -1 +1 @@
-path=check1 needs to be formatted
+paths=(local('check1'),) need to be formatted
```

Comment:
This error message needs to be fixed. It should say "('check',)" not "(local('check'),)".

Reply:
55a883d8ded98dee7e8d67ebd2ef0656c9bd2a71